<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertUI</class>
 <widget class="QWidget" name="InsertUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QWidget" name="caseWordWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{
	border-width:2px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
}
QWidget{
font: 75 12pt &quot;微软雅黑&quot;;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">.QLabel{
	border-right-width:2px;
	border-style:solid;	
	border-color: rgb(85, 170, 255);
}</string>
        </property>
        <property name="text">
         <string>关键字</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="caseWordLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLineEdit{
	border-right-width:0px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
    margin:0;
	padding:0;
	background-color: rgba(255, 255, 255,0);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="TypeWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{
	border-width:2px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
}
QWidget{
font: 75 12pt &quot;微软雅黑&quot;;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">.QLabel{
	border-right-width:2px;
	border-style:solid;	
	border-color: rgb(85, 170, 255);
}</string>
        </property>
        <property name="text">
         <string>类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="typeLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLineEdit{
	border-right-width:0px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
    margin:0;
	padding:0;
	background-color: rgba(255, 255, 255,0);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentEWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{
	border-width:2px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
}
QWidget{
font: 75 12pt &quot;微软雅黑&quot;;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">.QLabel{
	border-right-width:2px;
	border-style:solid;	
	border-color: rgb(85, 170, 255);
}</string>
        </property>
        <property name="text">
         <string>官方介绍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="contentEPlainTextEdit">
        <property name="styleSheet">
         <string notr="true">.QPlainTextEdit{
	border-right-width:0px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
    margin:0;
	padding:0;
	background-color: rgba(255, 255, 255,0);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentCWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{
	border-width:2px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
}
QWidget{
font: 75 12pt &quot;微软雅黑&quot;;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">.QLabel{
	border-right-width:2px;
	border-style:solid;	
	border-color: rgb(85, 170, 255);
}</string>
        </property>
        <property name="text">
         <string>翻译介绍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="contentCPlainTextEdit">
        <property name="styleSheet">
         <string notr="true">.QPlainTextEdit{
	border-right-width:0px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
    margin:0;
	padding:0;
	background-color: rgba(255, 255, 255,0);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="exampleWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{
	border-width:2px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
}
QWidget{
font: 75 12pt &quot;微软雅黑&quot;;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">.QLabel{
	border-right-width:2px;
	border-style:solid;	
	border-color: rgb(85, 170, 255);
}</string>
        </property>
        <property name="text">
         <string>示例</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="examplePlainTextEdit">
        <property name="styleSheet">
         <string notr="true">.QPlainTextEdit{
	border-right-width:0px;
	border-style:solid;
	border-color: rgb(85, 170, 255);
    margin:0;
	padding:0;
	background-color: rgba(255, 255, 255,0);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
	background-color: rgb(85, 170, 255);
	border-color: rgb(85, 170, 255);
	border-width:2px;
	border-style:solid;
	border-radius:5px;
	padding:5px;
	font: 75 9pt &quot;微软雅黑&quot;;
}
.QPushButton:hover{
	
	border-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
